lib/sqlalchemy/testing/reflection_fixture.py

```diff
@@ -0,0 +1,201 @@
+
+    def test_get_column_returns_computed(self):
+        insp = inspect(config.db)
+
+        cols = insp.get_columns("computed_default_table")
+        data = {c["name"]: c for c in cols}
+        for key in ("id", "normal", "with_default"):
+            is_true("computed" not in data[key])
+        compData = data["computed_col"]
+        is_true("computed" in compData)
+        is_true("sqltext" in compData["computed"])
+        actual = compData["computed"]["sqltext"]
+
+        eq_(self.to_sqltext("normal", "+", "42"), actual)
+        eq_("persisted" in compData["computed"], self.return_persisted)
+        if self.return_persisted:
+            eq_(compData["computed"]["persisted"], self.default_persisted)
+
+    def check_column(self, data, column, sqltext, persisted):
+        is_true("computed" in data[column])
+        compData = data[column]["computed"]
+        actual = compData["sqltext"]
+        eq_(sqltext, actual)
+        if self.return_persisted:
+            is_true("persisted" in compData)
+            is_(compData["persisted"], persisted)
+
+    def test_get_column_returns_persisted(self):
+        insp = inspect(config.db)
+
+        cols = insp.get_columns("computed_column_table")
+        data = {c["name"]: c for c in cols}
+
+        self.check_column(
+            data,
+            "computed_no_flag",
+            self.to_sqltext("normal", "+", "42"),
+            self.default_persisted,
+        )
+        if self.support_virtual:
+            self.check_column(
+                data,
+                "computed_virtual",
+                self.to_sqltext("normal", "+", "2"),
+                False,
+            )
+        if self.support_stored:
+            self.check_column(
+                data,
+                "computed_stored",
+                self.to_sqltext("normal", "-", "42"),
+                True,
+            )
+
+    def test_get_column_returns_persisted_with_schama(self):
+        insp = inspect(config.db)
+
+        cols = insp.get_columns(
+            "computed_column_table", schema=config.test_schema
+        )
+        data = {c["name"]: c for c in cols}
+
+        self.check_column(
+            data,
+            "computed_no_flag",
+            self.to_sqltext("normal", "/", "42"),
+            self.default_persisted,
+        )
+        if self.support_virtual:
+            self.check_column(
+                data,
+                "computed_virtual",
+                self.to_sqltext("normal", "/", "2"),
+                False,
+            )
+        if self.support_stored:
+            self.check_column(
+                data,
+                "computed_stored",
+                self.to_sqltext("normal", "*", "42"),
+                True,
+            )
+
+    def check_table_column(self, table, name, text, persisted):
+        is_true(name in table.columns)
+        col = table.columns[name]
+        is_not_(col.computed, None)
+        is_true(isinstance(col.computed, Computed))
```

Comment:
don't do this now but we should really have an "is_instance()" asserter in testing

Reply:
I have not checked to see if it exists.

Reply:
it doesn't.   this is more a note for me to do this when i ever rememebr

Reply:
there is `is_instance_of`